obs-web: fix em review data display

sethgerou-noaa opened the pull request.

neilriley-NOAA commented:
@seth. is there a way to populate the species name? I know the actual name is not submitted via in the EM data but it would be useful to see the name.

sethgerou-noaa commented:
yeah, looks like species name lookup/display used to be implemented, but that broke when we switched to number species codes - I've implemented a fix - will show up the next time I build obs-web.

neilriley-NOAA commented:
Great!. Thanks!
